app/services/search_service.rb

```diff
@@ -190,8 +191,47 @@ def recentPeople(srchString, tagName = nil)
       end
       doc = DocResult.fromLocationSearch(user.id, 'people_coordinates', user.path, user.username, 0, 0, user.lat, user.lon, blurred)
       sresult.addDoc(doc)
+      count += 1
+      if count == srchString.to_i then break end
     end
 
     sresult
   end
+
+  # Get users with matching name ordering by most recent activity
+  def getRecentProfiles
+    sresult = DocList.new
+
+    nodes = Node.all.order("changed DESC").limit(100).distinct
```

Comment:
I'm a little worried about the speed of this query. Could we collect up the `uid`s, then run:

```ruby
User.find_by(uid: uids.uniq).where(status: 1)
```

Then, we may also want to get recent edits, so what if we instead of doing `Node`, we searched `Revision` - that way we get both notes and wiki revisions?